src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -232,8 +232,8 @@ void OnOffServer::initOnOffServer(chip::EndpointId endpoint)
         // 0xff       Set the OnOff attribute to its previous value.
 
         // Initialize startUpOnOff to No action value 0xFE
-        uint8_t startUpOnOff = 0xFE;
-        EmberAfStatus status = Attributes::StartUpOnOff::Get(endpoint, &startUpOnOff);
+        chip::app::Clusters::OnOff::DlStartUpOnOff startUpOnOff = chip::app::Clusters::OnOff::DlStartUpOnOff::kNoAction;
```

Comment:
Please remove all the extra namespaces here and below.  This file has `using namespace chip::app::Clusters::OnOff;`

Reply:
Good point, will do so.